Microsoft.Toolkit.Uwp.DeveloperTools/VisualStudioToolsManifest.xml

```diff
@@ -1,8 +1,8 @@
 <FileList>
   <File Reference="Microsoft.Toolkit.Uwp.DeveloperTools.dll">
     <ToolboxItems UIFramework="UAP"
-        VSCategory="Windows Community Toolkit"
-        BlendCategory="Windows Community Toolkit">
+        VSCategory="Windows Toolkit | Design"
+        BlendCategory="Windows Toolkit | Design">
       <Item Type="Microsoft.Toolkit.Uwp.DeveloperTools.AlignmentGrid" />
```

Comment:
Not sure if we want continue to use "Windows Toolkit" or "Community Toolkit"? @michael-hawker

Reply:
We have to mention `Windows` somewhere atleast in the toolbox or else it would be confusing to people.

Reply:
We should not abridge the name of the Toolkit.

Reply:
If the name is too long, it'll get trimmed down due to the length of the toolbar.

It's best that we display controls by the Assembly container. So, including that, what would you propose the title name?

**Community Toolkit** or **Windows Toolkit**

Reply:
@michael-hawker @RosarioPulella 🔔

Reply:
@michael-hawker @RosarioPulella 🔔

Reply:
In that case why don't we put the most descriptive thing in front like "Developer Tools | Windows Community Toolkit"?

Reply:
The toolbox sorts the category in alphabetical order by default, so, if we put category first, then it would scatter when other toolkits are present.

Reply:
Is there a problem with us just putting them in one collective "Windows Community Toolkit" bucket like WinUI does?

Reply:
No, `WinUI` is a single assembly, so everything was put under single category. But Toolkit has many and having a separate category for each helps to see which is installed or not.

_For example, I saw clearly which controls were missing and **why**, when I separated them by category. I found that Markdown controls were missing and seeing that, I was able to fix it._

I have 3 suggestions for the naming...

1. We'll keep the `Windows Toolkit` now and rename it to `Community Toolkit` in the `WinUI` branch.

2. We'll change the name to `Community Toolkit` for both branches.

3. We'll change the name to `WCT <category> Controls` for both branches.

I prefer the **1st** suggestion as it aligns with our plans for `WinUI` branch. The other suggestions might be problematic since the controls will be displayed two times, one for each variant of the Toolkit, when using SxS (_Side by Side_).

Reply:
There shouldn't be a common case where you have both the UWP and WinUI 3 versions installed for controls. I'd be surprised if the Designer/Toolbox would display them as they're different tech stacks anyway.

I mean in general to a user, they don't know what they're missing. They'll just see the things from the Community Toolkit that they install. The default scenario is they install the uber 'Controls' package which has everything anyway. It's more advanced later to break-out into the sub-packages for what they need when they're preparing to deploy their app after it's been built.

I don't like the idea of us cluttering the Toolbox with a bunch of categories as then the developer has to open and poke at each one to see what they have access to (especially as some of our categories are more generalized based on dependencies).

I don't see a problem with us just having everything in one bucket (even if it makes it harder for us as the authors to audit, we shouldn't be optimizing for us but for our customer).

@chingucoding would love your thoughts here if you don't mind?

Reply:
I think it would be better to have a single category for this, as @michael-hawker mentioned, if we split it users would have to poke around. Having them inside a "WCT" category would be the easiest for our users and also a lot clearer than having multiple categories with subnames.